config/webpack.prod.js

```diff
@@ -31,6 +31,7 @@ const OfflinePlugin = require('offline-plugin');
  * Webpack Constants
  */
 const ENV = process.env.NODE_ENV = process.env.ENV = 'production';
+const OPENSHIFT_CONSOLE_URL = process.env.OPENSHIFT_CONSOLE_URL || 'https://console.starter-us-east-2.openshift.com/console/';
```

Comment:
or to say that another way - we should always use the only working environment, prod, unless someone wants to point at something else. Right now there's only the prod OSO anyways